### ISSUE TITLE
facebook_page_scraper.driver_utilities - critical - no posts were found!

### ISSUE THREAD

**Issue by ihabpalamino** [shaikhsajid1111/facebook_page_scraper#76]
hello @shaikhsajid1111 i am getting facebook_page_scraper.driver_utilities - critical - no posts were found! after scraping

**Comment by shaikhsajid1111** [comment]
Hey @ihabpalamino. I checked it is working. Maybe you're getting blocked somewhere. Try setting headless to `False` that may show you what is happening

**Comment by ihabpalamino** [comment]
yes it worked for me i have a question @shaikhsajid1111 for username i am not getting the same username i enter to scraped as the one display in my table 
![image](https://github.com/shaikhsajid1111/facebook_page_scraper/assets/86556941/e5f48bf5-8aa1-4613-af15-913477a0540e)
 for example if i am trying to scrap this page i enter elonmuskfandom but in my datatable iam getting this Elon Musk Fandom is  there any possibility to let it be elonmuskfandom

**Comment by neifion** [comment]
Same here, have set headless to False but still in vain :(

Has anyone had the same problem but figured it out? Thanks!
![Screenshot_2023-07-19_at_2_14_01_AM](https://github.com/shaikhsajid1111/facebook_page_scraper/assets/35992114/03641ba8-7586-47e3-8c7f-9a579321c2c4)
